src/transport/assembly_dg.hh

```diff
@@ -503,7 +503,7 @@ public:
                                 const Armor::array &point_list)
         {
             velocity.resize(point_list.size());
-            model_->velocity_field_ptr()->value_list(point_list, cell, velocity);
+            model_->data().flow_flux->value_list(point_list, cell, velocity);
```

Comment:
In order to perform scatter we can use:

`model_->data().flow_flux->get_field_fe()->local_to_ghost_data_scatter_begin()`